src/main/scala/nl.knaw.dans.easy.multideposit/parser/AudioVideoParser.scala

```diff
@@ -31,18 +31,21 @@ trait AudioVideoParser {
 
   def extractAudioVideo(rows: DepositRows, rowNum: Int, depositId: DepositId): Try[AudioVideo] = {
     Try {
-      ((springf: Option[Springfield], acc: Option[FileAccessRights.Value], avFiles: Set[AVFile]) => {
-        (springf, acc, avFiles) match {
-          case (None, _, fs) if fs.nonEmpty => Failure(ParseException(rowNum, "The column " +
+      ((springf: Option[Springfield], acc: Option[FileAccessRights.Value], playMode: Option[PlayMode.Value], avFiles: Set[AVFile]) => {
+        (springf, acc, playMode, avFiles) match {
+          case (None, _, _,  fs) if fs.nonEmpty => Failure(ParseException(rowNum, "The column " +
             "'AV_FILE' contains values, but the columns [SF_COLLECTION, SF_USER] do not"))
-          case (s, a, fs) => Try { AudioVideo(s, a, fs) }
+          case (s @ Some(_), a, None, fs) => Try { AudioVideo(s, a, Option(PlayMode.Continuous), fs) }
```

Comment:
yes, first of all this default was specified in the issue; second, this default only applies if the Springfield fields are provided (a.k.a. there is A/V material in it) and no SF_PLAY_MODE is given.